app/services/database_health_checker.rb

```diff
@@ -0,0 +1,26 @@
+module DatabaseHealthChecker
+  module_function
+
+  Summary = Struct.new(:healthy, :result) do
+    def as_json(*args)
+      to_h.as_json(*args)
+    end
+  end
+
+  # @return [Summary]
+  def check
+    Summary.new(true, simple_query)
+  rescue => err
+    Summary.new(false, err.message)
```

Comment:
I was thinking if something goes wrong with postgres and it tells us something we don't want to tell everyone. If we're confident that it won't, then no need to worry here.